pylti1p3/contrib/flask/request.py

```diff
@@ -0,0 +1,20 @@
+from pylti1p3.request import Request
+
+
+class FlaskRequest(Request):
+    _request = None
+    _cookies = []
+
+    def __init__(self, _request, _cookies, _session):
```

Comment:
underscore in the beginning of the variables looks like if this variables protected.
```
_request = None
_session = None
_cookies = []

def __init__(self, request, cookies, session):
    self.set_request(_request)
    self._session = session
    self._cookies = cookies
```
what do you think about change naming like i did it above?

Reply:
Sure, that was previously covering flask scope imports